GitTfs.sln

```diff
@@ -46,43 +48,71 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GitTfs.Setup", "GitTfs.Setu
 		{55C169E0-93CC-488C-9885-1D4EAF4EA236} = {55C169E0-93CC-488C-9885-1D4EAF4EA236}
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GitTfs.Vs2017", "GitTfs.Vs2017\GitTfs.Vs2017.csproj", "{7BEA3EC2-7111-4A5A-831F-2920C95EA356}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
```

Comment:
Can you remove the extra build configuration? It looks like it's mostly reusing project build configurations.

Having just an "x86" build configuration was important in the past, because some of the older TFS client libs are x86-only, so an AnyCPU build would sometimes run in 64-bit mode and then crash when loading the 32-bit DLLs, or something like that. At some point in the future we'll switch everything to be an "Any CPU" build, just not now.